supervisor/dbus/network/setting/__init__.py

```diff
@@ -234,6 +234,8 @@ async def reload(self):
                 interface_name=data[CONF_ATTR_CONNECTION].get(
                     CONF_ATTR_CONNECTION_INTERFACE_NAME
                 ),
+                mdns=data[CONF_ATTR_CONNECTION].get(CONF_ATTR_CONNECTION_MDNS),
+                llmnr=data[CONF_ATTR_CONNECTION].get(CONF_ATTR_CONNECTION_LLMNR),
```

Comment:
By testing, it seems when using the default (-1) the attribute is not even saved. So this get will return None, which then causes the API to show `llmnr: null` and `mdns: null`.

I haven't tried what happens when I try to update the network config, we probably have some safeguard to not actually write null back. But anyhow, we should just assume default if the property is missing, so:

```suggestion
                mdns=data[CONF_ATTR_CONNECTION].get(CONF_ATTR_CONNECTION_MDNS, -1),
                llmnr=data[CONF_ATTR_CONNECTION].get(CONF_ATTR_CONNECTION_LLMNR, -1),
```

We don't really use the NM settings specific enums in this code, but I think we maybe should use `MulticastDnsValue.DEFAULT.value` here.

Reply:
Good catch. Added `MulticastDnsValue.DEFAULT.value` here.